plugins/scanners/scanoss/src/test/kotlin/ScanOssScannerDirectoryTest.kt

```diff
@@ -103,7 +103,7 @@ class ScanOssScannerDirectoryTest : StringSpec({
                         Snippet(
                             99.0f,
                             TextLocation(
-                                "https://osskb.org/api/file_contents/871fb0c5188c2f620d9b997e225b0095",
+                                "examples/example.rules.kts", // This is the remote oss filepath
```

Comment:
Not sure if I had mentioned this before, but IMO the comment should be dropped for consistency.

Reply:
Yes, you did mention this before. I agree